kustomize/overlays/dev/data-plane/kustomization.yaml

```diff
@@ -2,12 +2,14 @@ resources:
   - https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard//etc/kubernetes/manifests/overlays/dev
 images:
   - name: quay.io/rhoas/cos-fleetshard-operator-camel
-    digest: sha256:e4a5578644dbeaf4fd0612e1c596553f3901d465d7f5201f5dbdd3c8fdf93005
+    digest: sha256:567da44f5dcbe3ed9e897682a83e642bbecc71de47d1013b3a928a7163971775
   - name: quay.io/rhoas/cos-fleetshard-operator-debezium
-    digest: sha256:cb2130c0fb46a6e097b41a95296a3863fdf38eed418416430c60e02b9646327b
+    digest: sha256:75c5e89b7f6dc314f0c7d50c49786dbb06d42fcfb7b16dd5546546a0edd5dc49
   - name: quay.io/rhoas/cos-fleetshard-sync
-    digest: sha256:8034d30827747d43e657bb534f31de7bc2a4efaed2ce5a5564a4244c5daadc66
+    digest: sha256:cd4d6d694a96ef101cd5a6d8680a019c7228983c765d3721574191a1a55303e1
   - name: quay.io/rhoas/cos-camel-k
     digest: sha256:f42213014a5a5164abddb71984b24864e3b73044df2ab94ce76f5aec42b2864c
   - name: quay.io/strimzi/operator
     digest: sha256:8b010c5fb8244b742a37f264feefc116d75bb2abd2d6895375cbeea250727b39
+  - name: "null"
```

Comment:
> @rinaldodev should we merge this one ?

I don't think so

Reply:
uh

Reply:
Haha, a good example demonstrating the value of PR reviews ;)